### PR TITLE
[ZEPPELIN-6042] Automate helium.json update

### PR DESCRIPTION
**Pull request by pythonstrup**
### What is this PR for?

This PR automate the update of helium.json using a Github actions script.

### What is the Jira issue?

[ZEPPELIN-6042](https://issues.apache.org/jira/browse/ZEPPELIN-6042)

### CODE REVIEW

**Comment by pythonstrup** [comment]
ping @jongyoul

File: .github/workflows/helium/.gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+
+node_modules
```
Comment (jongyoul) [inline .github/workflows/helium/.gitignore:7]: please add a new line at the end of the file. By the way, it's better to locate this file in the root directory.

File: .github/workflows/helium/lib/createHeliumPkgFile.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import Promise from "bluebird";
+import stringify from 'json-stringify-pretty-compact';
```
Comment (jongyoul) [inline .github/workflows/helium/lib/createHeliumPkgFile.js:2]: It looks like a different style. Could you please run prettier or linter?

File: .github/workflows/helium/lib/createHeliumPkgFile.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import {getMavenHandler} from "./getMavenArtifactInfo.js";
+import {fetchHelium} from "./fetchHeliumPkgInfo.js";
+import * as fs from "fs";
+import {getBasicMavenHandler} from "./getBasicMavenArtifactInfo.js";
+
+const pushPkgInfoToTmp = async function() {
+    // const response = await fetch('https://zeppelin.apache.org/helium.json')
+    // const data = await response.json();
+    // return data[0];
```
Comment (jongyoul) [inline .github/workflows/helium/lib/createHeliumPkgFile.js:11]:
```suggestion
```
If it's not necessary, could you please remove them?

**Comment by jongyoul** [comment]
Thank you for your contribution. LGTM except for some minor comments.

File: .github/workflows/helium/.gitignore (hunk 1 above)
Comment (pythonstrup) [reply .github/workflows/helium/.gitignore:7]: Sure, I'll try placing it in the root directory.

File: .github/workflows/helium/lib/createHeliumPkgFile.js (hunk 2 above)
Comment (pythonstrup) [reply .github/workflows/helium/lib/createHeliumPkgFile.js:2]: Okay. I'll run the linter.

File: .github/workflows/helium/lib/createHeliumPkgFile.js (hunk 3 above)
Comment (pythonstrup) [reply .github/workflows/helium/lib/createHeliumPkgFile.js:11]: I miss that. I'll remove it and push again.

**Comment by pythonstrup** [comment]
@jongyoul 
Thank you for review. I've fixed everything and pushed it.

**Review by jongyoul — APPROVED** [review]

**Pull request merged**